includes/admin/settings/class-wc-settings-general.php

```diff
@@ -75,6 +75,15 @@ public function get_settings() {
 			),
 
 			array(
+				'title'   => __( 'Sell to all Countries except', 'woocommerce' ),
```

Comment:
I'd reword these to 'Exclude specific countries", and move it after the 'sell to' option.